src/contracts/libraries/GPv2Transfer.sol

```diff
@@ -32,10 +32,17 @@ library GPv2Transfer {
     /// This method is used for transferring fees to the settlement contract
     /// when settling a single order directly with Balancer.
     ///
+    /// Note that this method is subtly different that `transferFromAccounts`
+    /// with a single transfer with respect to how it deals with internal
+    /// balances. Specifically, this method will perform an **internal balance
+    /// transfer instead of an intenal balance withdrawal** for trades that
```

Comment:
Maybe also document the recipient of the internal transfer?

```suggestion
    /// balances. Specifically, this method will perform an **internal balance
    /// transfer to the settlement contract instead of a withdrawal to the external balance of the settlement contract** for trades that
```